File: year2020/month12/day1224/ms_1224.cpp
#include <iostream>
#include <string>

using namespace std;

string solution(string phone_number);

int main() {
	string pn;

	getline(cin, pn);

	if (4 <= pn.length() && pn.length() <= 20) {
		cout << solution(pn) << endl;
	}
}

string solution(string phone_number) {
	int len = phone_number.length();
	string answer = "";

	for (int i = 0; i < len - 4; i++) {
		answer += "*";
	}

	answer += phone_number.substr(len - 4, 4);

	return answer;
}

File: year2020/month12/day1224/ms_1224.cs
using System;

namespace algorithm_129
{
    class ms_1224
    {
        static string solution(string phone_number) 
        {
            int len = phone_number.Length;
            string answer = "";

            for(int i = 0; i < len - 4; i++) 
            {
                answer += "*";
            }

            answer += phone_number.Substring(len - 4, 4);

            return answer;
        }

        static void Main(string[] args)
        {
            string pn = Console.ReadLine();
            int len = pn.Length;

            if(4 <= len && len <= 20)
            {
                Console.WriteLine(solution(pn));
            }
        }
    }
}
